test/tbb/test_global_control.cpp

```diff
@@ -272,3 +273,30 @@ TEST_CASE("test concurrent task_scheduler_handle destruction") {
     stop = true;
     thr1.join();
 }
+
+//! \brief \ref regression
+TEST_CASE("Thread should sleep when soft_limit is zero") {
```

Comment:
Suggesting more blackbox approach for naming this test.
```suggestion
TEST_CASE("Test worker threads remain inactive in enforced serial execution mode") {
```